Tests/Controller/ControllerTest.php

```diff
@@ -109,11 +109,17 @@ public static function dataProviderForTestGenerateWithCallback()
         );
     }
 
-    public function testGenerateWithInvalidCallback()
+    /**
+     * @testWith ["(function xss(x) {evil()})"]
+     *           [["foo", "bar"]]
+     *
+     * @group legacy
```

Comment:
with Symfony 5.1+ this will be fixed already:

```
Remaining indirect deprecation notices (1)

  1x: Since symfony/http-foundation 5.1: Retrieving a non-string value from "Symfony\Component\HttpFoundation\InputBag::get()" is deprecated, and will throw a "Symfony\Component\HttpFoundation\Exception\BadRequestException" exception in Symfony 6.0, use "Symfony\Component\HttpFoundation\InputBag::all($key)" instead.
    1x in ControllerTest::testGenerateWithInvalidCallback from FOS\JsRoutingBundle\Tests\Controller